sklearn/datasets/_base.py

```diff
@@ -52,6 +52,11 @@ def get_data_home(data_home=None) -> str:
     data_home : str, default=None
         The path to scikit-learn data directory. If `None`, the default path
         is `~/sklearn_learn_data`.
+
+    Returns
+    -------
+    data_home: str
+            The path of the scikit-learn data dir.
```

Comment:
```suggestion
            Path of the scikit-learn data directory.
```

Reply:
Based on the other docstrings, I guess we can keep the text unchanged, unintended the text a bit and keep it verbose, like what @jmloyola proposes. 
```suggestion
        The path of the scikit-learn data directory.
```